tests/contrib/django/test_django_snapshots.py

```diff
@@ -24,57 +14,27 @@ def test_urlpatterns_include(client):
     assert client.get("/include/test/").status_code == 200
 
 
-@pytest.mark.skipif(django.VERSION >= (1, 9), reason="")
-@snapshot()
-def test_middleware_trace_callable_view_18x(client):
-    # ensures that the internals are properly traced when using callable views
-    assert client.get("/feed-view/").status_code == 200
-
-
-@pytest.mark.skipif(not (django.VERSION >= (1, 9) and django.VERSION < (1, 12)), reason="")
-@snapshot()
-def test_middleware_trace_callable_view_111x(client):
-    # ensures that the internals are properly traced when using callable views
-    assert client.get("/feed-view/").status_code == 200
-
-
-@pytest.mark.skipif(not (django.VERSION > (1, 12) and django.VERSION < (2, 2)), reason="")
-@snapshot()
-def test_middleware_trace_callable_view_21x(client):
-    # ensures that the internals are properly traced when using callable views
-    assert client.get("/feed-view/").status_code == 200
-
-
-@pytest.mark.skipif(django.VERSION < (2, 2), reason="")
-@snapshot()
+@snapshot(
+    variants={
+        "18x": django.VERSION < (1, 9),
+        "111x": (1, 9) <= django.VERSION < (1, 12),
+        "21x": (1, 12) < django.VERSION < (2, 2),
+        "": django.VERSION >= (2, 2),
```

Comment:
is the `""` case needed? if the others all fail, we fallback to just `token` right?

Reply:
Right, that's definitely true. This is a slightly verbose approach I took but I think it's helpful to be explicit.

Reply:
I'm a fan of being explicit until it becomes apparent that we get something meaningful from an implicit default/fallback.

Reply:
yeah, in hindsight being explicit is probably better since otherwise reading the code you might think the `""` case is actually running